tests/test-screen-changes.lua

```diff
@@ -93,9 +93,7 @@ local steps = {
         assert(test_client2.screen == real_screen, test_client2.screen)
 
         collectgarbage("collect")
-        if #fake_screen == 0 then
-            return true
-        end
+        return #fake_screen == 0
```

Comment:
The test runner (`tests/_runner.lua`) does the following after a step ran:
- If it returned true, it continues with the next step
- If it returns false, the whole test is considered to have failed
- If it returns nil, this step is retried after 0.1 seconds, but only 5 times. Then the test is considered to have failed.

So, before this commit, the garbage collector had up to 5 calls to `collectgarbage("collect")` time to get rid of the old screen. Now the screen must be removed on the first collection cycle.

I guess this change is the reason why the Travis build failed. Also, this cannot stall, since the test runner only tries up to five times and then gives up.

Please remove this commit.

Reply:
The Travis build was failing even before I introduced this commit, due to (I think) references to screen being held in the new font cache. But thanks for the explanation, I'll remove the commit.